### PR TITLE
[MU4] Removed old TODO

### PR DESCRIPTION
**Pull request by RomanPudashkin**

### CODE REVIEW

File: src/engraving/libmscore/segment.cpp [hunk 1]
```diff
@@ -1655,8 +1655,7 @@ Spanner* Segment::firstSpanner(int activeStaff)
             if (s->startSegment() == this) {
                 if (e->staffIdx() == activeStaff) {
                     return s;
-                }
-                else if (e->isMeasure() && activeStaff == 0) {
+                } else if (e->isMeasure() && activeStaff == 0) {
                     return s;
```
Comment (cbjeukendrup) [inline src/engraving/libmscore/segment.cpp]: Could probably also be 
```cpp
    if (e->staffIdx() == activeStaff 
        || (e->isMeasure() && activeStaff == 0)) {
        return s;
    }
```

(Same for line 1683)

Comment (RomanPudashkin) [reply src/engraving/libmscore/segment.cpp]: fixed

File: src/engraving/libmscore/barline.cpp [hunk 2]
```diff
@@ -1268,43 +1130,7 @@ void BarLine::endEditDrag(EditData& ed)
...
-            if (newSpanTo < MIN_BARLINE_SPAN_FROMTO) {
-                newSpanTo = MIN_BARLINE_SPAN_FROMTO;
-            }
-            if (newSpanTo > maxTo) {
-                newSpanTo = maxTo;
-            }
-        }
-    }
-#endif
 //      else {                              // if coarse dragging
```
Comment (Eism) [inline src/engraving/libmscore/barline.cpp:1134]: need to remove too

File: src/engraving/libmscore/chordlist.cpp [hunk 3]
```diff
@@ -1022,22 +967,6 @@ bool ParsedChord::parse(const QString& s, const ChordList* cl, bool syntaxOnly,
...
-                    } else if (tok2L == "2") {
-                        _xmlKind = "suspended-second";
-                    }
-                    _xmlText = tok1 + tok2;
-                } else {
-                    _xmlDegrees += "sub3";
-                    tok1L = "add";
-                }
-#else
 // enable this code to export 7sus4 as suspended-fourth,add7 rather than dominant,sub3,add4 (similar for 9sus4, 13sus4)
```
Comment (Eism) [inline src/engraving/libmscore/chordlist.cpp:970]: need to remove too

Comment (RomanPudashkin) [reply src/engraving/libmscore/chordlist.cpp]: removed

File: src/engraving/libmscore/barline.cpp (hunk 2 above)
Comment (RomanPudashkin) [reply src/engraving/libmscore/barline.cpp]: removed

**Review by igorkorsukov — APPROVED** [review]

**Pull request merged**